src/main/features/core/tray.js

```diff
@@ -180,5 +180,50 @@ Emitter.on('audiooutput:list', (event, devices) => {
   setContextMenu();
 });
 
+// Change the Icon if its playing
+Emitter.on('playback:isPlaying', () => {
```

Comment:
yep, if i use:
`PlaybackAPI.on('change:state', (event, state) => {
  if (state === true) {
    if (process.platform === 'darwin') {
      appIcon.setImage(path.resolve(`${trayPlayingPath}/macTemplate.png`));
    } else if (WindowManager.getWindowManagerGDMName() === 'kde-plasma') {
      // TODO: Change this back to ico when electron supports it on linux
      // appIcon = new Tray(path.resolve(`${__dirname}/../../../assets/img/main.ico`));
      appIcon.setImage(path.resolve(`${trayPlayingPath}/main_tray_white_s.png`));
    } else if (process.platform === 'linux') {
      appIcon.setImage(path.resolve(`${trayPlayingPath}/main_tray_white_s.png`));
    } else {
      appIcon.setImage(path.resolve(`${trayPlayingPath}/main_tray_s.png`));
    }
  } else {
    if (process.platform === 'darwin') {
      appIcon.setImage(path.resolve(`${trayPausedPath}/macTemplate.png`));
    } else if (WindowManager.getWindowManagerGDMName() === 'kde-plasma') {
      // TODO: Change this back to ico when electron supports it on linux
      // appIcon = new Tray(path.resolve(`${__dirname}/../../../assets/img/main.ico`));
      appIcon.setImage(path.resolve(`${trayPausedPath}/main_tray_white_s.png`));
    } else if (process.platform === 'linux') {
      appIcon.setImage(path.resolve(`${trayPausedPath}/main_tray_white_s.png`));
    } else {
      appIcon.setImage(path.resolve(`${trayPausedPath}/main_tray_s.png`));
    }
  }
});`

and start it, it will initialize with the paused button. not sure if you want it this way, but it bothers me  abit